tut/installing-appliance.md

```diff
@@ -66,12 +66,7 @@ Run the following command to install the `anbox-cloud-appliance` snap, which han
 <a name="additional-tools"></a>
 ### 5. Install additional tools
```

Comment:
Should we keep it in its own subsection or should we merge two subsections?

This may be better left to another PR if you want to keep it small in scope, though.

Reply:
No, I think it can stay in a separate section because it is a tutorial and it is useful for readers to understand that AMC is an additional tool we are installing on top of the appliance. Also, the additional tools sub section is referred in the getting started tutorial as a prereq.